docs/resources/User.md

```diff
@@ -43,6 +43,7 @@ There are other rules and restrictions not shared here for the sake of spam and
 | premium_type?      | integer   | the [type of Nitro subscription](#DOCS_RESOURCES_USER/user-object-premium-types) on a user's account | identify              |
 | public_flags?      | integer   | the public [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                  | identify              |
 | avatar_decoration? | ?string   | the user's [avatar decoration hash](#DOCS_REFERENCE/image-formatting)                                | identify              |
+| clan?              | ?UserClan | the user's [clan data](#DOCS_RESOURCES_USER/user-object-user-clan)                                   | identify              |
```

Comment:
```suggestion
| clan?              | ?[user clan](#DOCS_RESOURCES_USER/user-object-user-clan) object | the user's clan data                               | identify              |
```
Leaving the table formatting aside (which currently this would probably break) the corresponding object should be mentioned in the `type` column entry and not the `description` to stay consistent across the docs...